test/default.nix

```diff
@@ -8,6 +8,11 @@ let
   hn-rust-fmt-check \
   && hn-rust-clippy \
   && cargo test
+
+  cargo test \
+    --manifest-path="crates/holochain_serialized_bytes/Cargo.toml" \
+    --features="trace" \
```

Comment:
just as an option, you can do `--all-features` - might not have to keep adding lines like this for each new feature.